lib/std/fs.zig

```diff
@@ -226,6 +226,2467 @@ pub fn renameW(old_dir: Dir, old_sub_path_w: []const u16, new_dir: Dir, new_sub_
     return os.renameatW(old_dir.fd, old_sub_path_w, new_dir.fd, new_sub_path_w);
 }
 
+/// A directory that can be iterated. It is *NOT* legal to initialize this with a regular `Dir`
```

Comment:
I'm not sure why these are +2,451 additions are needed or shown.

Reply:
It might be from me rebasing incorrectly

Reply:
It is definitely from me rebasing incorrectly

Reply:
Fixed now, sorry about that. I got too trigger happy with the squashing, apparently